doc/user_doc/vic_installation/vch_installer_options.md

```diff
@@ -269,380 +374,395 @@ To specify a container network, you provide the name of a distributed port group
 
-<a name="datastore"></a>
-## Datastore Options ##
-The `vic-machine` utility allows you to specify the datastores in which to store container image files, the files for the virtual container host appliance, container VM files, and container volumes. 
+- Add optional information to auto-generated trusted TLS certificates by specifying the `--tls-ca`, `--certificate-key-size`, and `--organization` options.
+- Use custom trusted TLS certificates by using the `--cert` and `--key` options.
+- Disable TLS authentication completely by using the `--no-tls` option.
 
-- vSphere Integrated Containers Engine fully supports VMware Virtual SAN datastores. 
-- vSphere Integrated Containers Engine supports all alphanumeric characters, hyphens, and underscores in datastore paths and datastore names, but no other special characters.
-- If you specify different datastores in the different datastore options, and if no single host in a cluster can access all of those datastores, `vic-machine create` fails with an error.<pre>No single host can access all of the requested datastores. 
-Installation cannot continue.</pre>
-- If you specify different datastores in the different datastore options, and if only one host in a cluster can access all of them, `vic-machine create` succeeds with a warning.<pre>Only one host can access all of the image/container/volume datastores. 
-This may be a point of contention/performance degradation and HA/DRS 
-may not work as intended.</pre> 
+### `--tls-ca` ###
 
-### `image-store` ###
+Short name: `--ca`
 
-See [image-store](#image) in the section on mandatory options.
+Certificate Authority (CA) files to use to verify Docker client certificates. Specify the `--tls-ca` option if your certificates are validated by a CA that is not commonly recognized. Specify the `--tls-ca` option multiple times to specify multiple CA files. 
 
-### `container-store` ###
+<pre>--tls-ca <i>path_to_ca_file</i></pre>
 
-Short name: `--cs`
+### `--certificate-key-size` ###
 
-The datastore in which to store container VM files. When you run a container, container VM files are stored in folders at the top level of the designated datastore. If multiple virtual container hosts use the same container store, all of the container VM files appear at the top level of the container store. You cannot currently designate a specific datastore folder for the VM files of the containers that run in a particular virtual container host.
+Short name: `--ksz`
 
-If you are deploying the virtual container host to a vCenter Server cluster, the datastore that you designate in the `container-store` option must be shared by at least two ESXi hosts in the cluster. Using non-shared datastores is possible, but limits the use of vSphere features such as DRS and High Availability.
+The size of the key for `vic-machine create` to use when it creates auto-generated trusted certificates. If not specified, `vic-machine create` creates keys with default size of 2048 bits.
```

Comment:
Updated text and example:

---
If not specified, `vic-machine create` creates keys with default size of 2048 bits. It is not recommended to use key sizes of less than 2048 bits.

<pre>--certificate-key-size 3072</pre>

---